Web/src/components/games/BalloonShake/GameDisplay.tsx

```diff
@@ -1,66 +1,63 @@
 import React from 'react';
 import styled from 'styled-components';
-import { ReactComponent as BalloonSVG } from 'svg/balloon.svg';
+import TimerDisplay from 'components/games/TimerDisplay';
+import { ReactComponent as Balloon } from 'svg/balloon.svg';
 
 interface IProps {
   secondsLeft: number;
   count: number;
 }
 
 const GameContainer = styled.div`
-  background: var(--pale-yellow);
+  background: var(--pink);
   overflow: hidden;
-  height: 100vh;
-  width: 100vw;
+  height: ${window.innerHeight}px;
```

Comment:
wow hax

Reply:
Haha a scrolly page is pretty annoying on phone browser :c